src/stores/businessStore.ts

```diff
@@ -12,6 +12,8 @@ export const useBusinessStore = defineStore('business', {
     businessInfo: {
       adminFreeze: null,
       allowedActions: null,
+      arMaxDate: null,
+      arMinDate: null,
```

Comment:
These are no longer optional in the interface.

Below, I provide getters to these properties.